pkg/ocm/ocm.go

```diff
@@ -43,14 +43,27 @@ func New(ocmConfigFile string) (Client, error) {
 	var err error
 	client := Client{}
 
-	cfg, err := newConfigFromFile(ocmConfigFile)
-	if err != nil {
-		return client, fmt.Errorf("failed to load config file: %w", err)
+	ocmClientID, hasOcmClientID := os.LookupEnv("OCM_CLIENT_ID")
+	ocmClientSecret, hasOcmClientSecret := os.LookupEnv("OCM_CLIENT_SECRET")
+	ocmURL, hasOcmURL := os.LookupEnv("OCM_URL")
+
+	if !hasOcmClientID || !hasOcmClientSecret || !hasOcmURL {
+		cfg, err := newConfigFromFile(ocmConfigFile)
+		if err != nil {
+			return client, fmt.Errorf("failed to load config file: %w", err)
+		}
+
+		client.conn, err = cfg.Connection()
+		if err != nil {
+			return client, fmt.Errorf("failed to create new OCM connection from config file: %w", err)
+		}
+
+		return client, nil
 	}
 
-	client.conn, err = cfg.Connection()
+	client.conn, err = sdk.NewConnectionBuilder().URL(ocmURL).Client(ocmClientID, ocmClientSecret).Insecure(true).Build()
```

Comment:
Why insecure true?

Reply:
That's a suggestion from the Slack Channel :thinking: I didn't ask for a reason. Are the OCM certificates self-signed? If not, we can set Insecure to False

Reply:
Makes sense, should we card this for later aswell?
If we can write on each possible security hole the reasoning, it makes auditing focus on things that we can really fix

Reply:
I have checked api.openshift.com and api.stage.openshift.com.. both have valid certificates. I think we can set insecure to false